ckanext/ontario_theme/templates/internal/snippets/home_breadcrumb_item.html

```diff
@@ -0,0 +1,6 @@
+{# Used to insert the home icon into a breadcrumb #}
```

Comment:
Will we be inserting the home icon?

Reply:
No, the home icon was removed. That was a legacy comment, I'll delete it, thanks!